src/components/sidebar/sidebar.tsx

```diff
@@ -50,12 +59,20 @@ export default function Sidebar({ className, ...rest }: SidebarProps) {
         <EditItems />
         <EditTiers />
       </div>
-      <button
-        className="btn btn-sm btn-danger btn-clear w-100 mt-2"
-        onClick={handleClear}
-      >
-        Clear
-      </button>
+      <div className="edit-row mt-1">
+        <button
+          className="btn btn-sm btn-danger btn-clear w-100 mt-2"
+          onClick={handleClear}
+        >
+          Clear
+        </button>
+        <button
+          className="btn btn-sm w-100 mt-2"
+          style={{ backgroundColor: "#6a1717" }}
+          onClick={handleReset}>
```

Comment:
It would be better to over write warning color in scss file to this value. You can then use btn-warning class.